app/src/organisms/RobotSettingsCalibration/index.tsx

```diff
@@ -120,6 +122,11 @@ export function RobotSettingsCalibration({
 
   const pipetteOffsetCalibrations = useAllPipetteOffsetCalibrationsQuery().data
     ?.data
+  const { data: attachedInstruments } = useInstrumentsQuery()
```

Comment:
Gripper Calibration section won't fetch the data after calibration.
But the thing is that useInstrumentsQuery includes pipette data.
Maybe maybe we can add if statement with `isOnDevice` to line 250-252  to execute fetch when a robot is an OT-2 🤔 

```suggestion
  const { data: attachedInstruments } = useInstrumentsQuery({refetchInterval: CALS_FETCH_MS})
```